.changes/unreleased/ENHANCEMENTS-20250909-163330.yaml

```
@@ -0,0 +1,6 @@
+kind: ENHANCEMENTS
+body: Configuration option `terraform.mcp.server.enabled` to enable or disable HashiCorp Terraform MCP Server integration
```

```suggestion
body: Configuration option `terraform.mcp.server.enable` to enable or disable HashiCorp Terraform MCP Server integration
```

I suggest using `terraform.mcp.server.enable` as there is already `terraform.languageServer.enable`. When the user sees this in the Settings UI, it will look like `[ ] Enable` instead of a confusing `[ ]  Enabled`.